oui-ui-core/Makefile

```diff
@@ -64,6 +64,8 @@ endef
 
 define Package/oui-ui-core/postinst
 #!/bin/sh
+[ -z "$${IPKG_INSTROOT}" ]; && exit 0
```

Comment:
@zhaojh329 sorry I have found a small mistake from my side the `;` is not needed!
That was a carelessness on my part sorry!

Reply:
That is my fix https://github.com/zhaojh329/oui/pull/49 for that